tests/python/contrib/test_clml/menangerie.py

```diff
@@ -0,0 +1,4179 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I see only resnet50, resnet50_16, mobilenet, mobilenet_16 are used for test cases.
You may remove other network definitions here.

Reply:
Removed the unused code!